### PR TITLE
Oy2 9914 test

### PR DESCRIPTION
**Pull request by jdinh8124**
Completed automation testing for APMCH

### CODE REVIEW

File: .github/workflows/deploy.yml [hunk 1]
```diff
@@ -289,13 +287,16 @@ jobs:
       matrix:
         containers:
           [
+            ADDCH.spec.js,
+            APMCH.spec.js,
```
Comment (jphechter) [inline .github/workflows/deploy.yml:291]: @daniel-belcher When we me merge `master` back into this work, we need to make sure that we don't loose the automation tests.

File: tests/cypress/cypress/integration/measures/APMCH.spec.js [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+    );
+    cy.get(
+      '[data-cy="At least one Performance Measure Numerator, Denominator, and Rate must be completed"] > .chakra-text'
+    ).should(
+      "have.text",
+      "At least one Performance Measure Numerator, Denominator, and Rate must be completed"
+    );
+  });
+
+  it("at least one dnr set if reporting and measurement spec or error.", () => {
```
Comment (jphechter) [inline tests/cypress/cypress/integration/measures/APMCH.spec.js:46]:
```suggestion
  it("at least one NDR set if reporting and measurement spec or error.", () => {
```

**Review by jphechter — APPROVED** [review]
Spelling in one prompt, otherwise LGTM 👍

File: .github/workflows/deploy.yml (hunk 1 above)
Comment (daniel-belcher) [reply .github/workflows/deploy.yml:291]: definitely keeping an eye on this

**Pull request merged**